add web modeler configuration draft (#4332)
camunda/camunda-docs: docs/components/modeler/web-modeler/play-your-process.md

To use Play, open a BPMN diagram and click the **Play** tab. Read the [limitations and availability section](#limitations-and-availability) if this section is missing.

> In Self-Managed, you are prompted to select from any clusters defined in your Web Modeler [configuration](/self-managed/modeler/web-modeler/configuration/configuration.md#clusters). The Camunda 8 Docker Compose distribution provides one cluster configured by default. If no configuration is found, you are prompted to enter your cluster details [manually](#use-play-with-camunda-self-managed).

Reviewer: ```suggestion
In Self-Managed, you are prompted to select from the clusters defined in your Web Modeler [configuration](/self-managed/modeler/web-modeler/configuration/configuration.md#clusters). The Camunda 8 Docker Compose distribution provides one cluster configured by default. If no configuration is found, you are prompted to [manually enter your cluster details](#use-play-with-camunda-self-managed).
```
Minor suggestions

> A Play environment is then started that utilizes your selected development cluster in SaaS, or the specified cluster in a Self-Managed setup.

Reviewer: ```suggestion
A Play environment then starts, using your selected development cluster in SaaS, or the specified cluster in a Self-Managed setup.
```
Suggestion, feel fee to ignore

## Use Play with Camunda Self-Managed

> After selecting the **Play** tab in Self-Managed, you are prompted to select from any clusters defined in your Web Modeler [configuration](/self-managed/modeler/web-modeler/configuration/configuration.md#clusters). The Camunda 8 Docker Compose distribution provides one cluster configured by default.

Reviewer: ```suggestion
After selecting the **Play** tab in Self-Managed, you are prompted to select from the clusters defined in your Web Modeler [configuration](/self-managed/modeler/web-modeler/configuration/configuration.md#clusters). The Camunda 8 Docker Compose distribution provides one cluster configured by default.
```
Suggestion

> If no cluster is configured, Web Modeler will request the following cluster details to use for deployment:

Reviewer: ```suggestion
If no cluster is configured, Web Modeler requests the following cluster details to use for deployment:
```
